src/servises/formatString.ts

```diff
@@ -0,0 +1,3 @@
+export const formatString = (str: string): string => {
+  return str.split(' ').join(' ');
```

Comment:
The current implementation of `formatString` does not change the input string. It splits the string by spaces and joins it back, which results in the same string. Consider implementing a meaningful transformation if required by the task.